certbot/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 * Updated Apache/NGINX TLS configs to document contents are based on ssl-config.mozilla.org
+* Updated Windows installer to be signed and trusted in Windows 
```

Comment:
Oops, we're now in 1.29.0, so let's just move this to the latest section after merging in master.